### PR TITLE
Adds lines to brain damage

### PR DESCRIPTION
**Pull request by harmonyn**
lines confirmed not ooc in ic

justification: because I laughed

### CODE REVIEW

**Comment by ExcessiveUseOfCobblestone** [comment]
The sniff test for these phrases seem to be conducted by someone who lives by a barn, particularly near the location where they keep the manure.

File: strings/traumas.json [hunk 1]
```diff
@@ -63,7 +63,8 @@
         "@pick(semicolon)wtf??????????? @pick(bug)",
         "@pick(semicolon)i ran into the supermattre ten i dsappeard @pick(bug)",
         "DON'T EVER TUCH ME",
-        "@pick(semicolon)GIVE ME FREE ROBUX PLEASE JUST ENOUGH FOR SHIRT AND PANTS"
+        "@pick(semicolon)GIVE ME FREE ROBUX PLEASE JUST ENOUGH FOR SHIRT AND PANTS",
+		"its gonna be like 9/11 all over again but insitaead is gonna be a bitc"
```
Comment (ninjanomnom) [inline strings/traumas.json]: This file uses space indentation not tab indentation

**Review by ninjanomnom — DISMISSED** [review]

File: strings/traumas.json (hunk 1 above)
Comment (ma44) [reply strings/traumas.json]: **heresy**

File: strings/traumas.json [hunk 2]
```diff
@@ -129,7 +131,10 @@
         "treaitors",
         "sheadow lings",
         "abdoocters",
-        "revinent"
+        "revinent",
+        "deval",
+        "deth squads",
```
Comment (ShizCalev) [inline strings/traumas.json]: squids

**Review by Cyberboss — APPROVED** [review]

**Pull request merged**